tests/codebuild/update_tests.sh

```diff
@@ -9,12 +9,6 @@ source inject_tests.sh
 function run_update_canary_tests
 {
   local crd_namespace="$1"
-
-  echo "Injecting variables into tests"
-  inject_all_variables
```

Comment:
Inject should stay here. Eventually when we add update tests for canaries, they'll run here first. The integration tests always run the `run_update_canary_tests` function too.